### DISCUSSION TITLE
Where to find out why the logsnever sent from fluentd?

### DISCUSSION THREAD

**Discussion by frit0-rb** [kube-logging/logging-operator#1727] [Q&A] [answered]
Good afternoon,

I got a cluster RKE2 kubernetes with 4.5.1 logging-operator.

I have a logging which recollect logs from some apps with Flows and send via ClusterOuput to Splunk.

The recollection looks good but the problem thar logs never sent to Splunk server.

I'm wondering where can I find why this logs never sent to Splunk server? Are there any logs or something else in fluentd or fluentbit to see that?

Thanks in advance

**Comment by pepov** [comment, marked as answer]
You can raise the [log level](https://kube-logging.dev/docs/operation/troubleshooting/fluentd/#set-fluentd-log-level) to trace to see what happens inside fluentd. 

You can also use the [stdout filter](https://kube-logging.dev/docs/configuration/plugins/filters/stdout/) like printf and check your flows one by one
